Services/TMS/Booking/classes/class.ilTMSSuperiorBookingGUI.php

```diff
@@ -0,0 +1,371 @@
+<?php
+/**
+ * cat-tms-patch start
+ */
+
+use ILIAS\TMS\Booking;
+
+require_once("Services/Form/classes/class.ilPropertyFormGUI.php");
+require_once("Services/TMS/Booking/classes/class.ilTMSBookingPlayerStateDB.php");
+
+/**
+ * Displays the TMS superior booking
+ *
+ * @author Richard Klees <richard.klees@concepts-and-training.de>
+ */
+class ilTMSSuperiorBookingGUI extends Booking\Player {
```

Comment:
Bitte führe eine gemeinsame Basisklasse für die beiden BookingGUIs ein, da kann dann der duplizierte Code drin leben. In erster Instanz muss dann in den beiden spezialisierten BookingGUIs nur `Booking\Player::getComponentClass` entsprechend überschrieben werden.